.github/workflows/main.yml

```diff
@@ -43,6 +43,11 @@ jobs:
       env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Upload Jest coverage to Github Artifacts
```

Comment:
@vickywane should this name value be updated to "Upload combined test coverage report ..." ?

Reply:
Yes, i have renamed this job.